src/rpc/mining.cpp

```diff
@@ -790,6 +790,12 @@ UniValue estimatefee(const JSONRPCRequest& request)
             + HelpExampleCli("estimatefee", "6")
             );
 
+    if (!IsDeprecatedRPCEnabled(std::string("estimatefee"))){
```

Comment:
I initially had this before the help if statement, but the way that help is implemented in `server.cpp` means that calling help without any arguments calls into every RPC method. Adding the `if (!IsDeprecatedRPCEnabled` statement before the help if statement breaks that.

You're right that no explicit `std::string()` is required. Removing.